docs/root/api-docs/xds_protocol.rst

```diff
@@ -678,6 +678,14 @@ server does not provide EDS/RDS responses, Envoy will not initialize
 itself during the initialization phase and the updates sent via CDS/LDS
 will not take effect until EDS/RDS responses are supplied.
 
+.. note::
+
+   Envoy specific implementation note: Warming of ``Cluster`` is completed only
+   when a new ``ClusterLoadAssignment`` response is supplied by management server
+   even though there is no change in endpoints. However, If Envoy is already aware of
+   ``RouteConfiguration`` referenced by ``Listener``, it will use it otherwise Envoy will
+   initiate a new RDS request with the new route name referenced by ``Listener``.
```

Comment:
I can't parse this. Is this related to Cluster or unrelated and a different note about warming? Can you clarify and if it's different move it to a separate note?

Reply:
It is about listener warming with route. It is under same resource warming category.

Reply:
OK then I don't understand this and how it relates to the cluster part. Can you clarify? Thanks.

Reply:
It is not related to cluster warming. It is related to resource warming only but talks about how listeners are warmed.

Reply:
Then please split it out into a separate note and/or make it a clearly separate statement. I'm having a hard time parsing what this means. Thank you!

Reply:
I clarified it. PTAL